variables.tf

```diff
@@ -66,6 +68,34 @@ variable "unique_writer_identity" {
   default     = null
 }
 
+# FOLDER
+
+variable "folder" {
+  type        = string
+  description = "(Optional) The ID of the folder to create the sink in. If omitted, no folder logging sink is created. If provided along with `var.project`, only the project logging sink is created."
+  default     = null
+}
+
+variable "include_folder_children" {
+  type        = bool
+  description = "(Optional) Whether or not to include children folders in the sink export. If true, logs associated with child projects are also exported; otherwise only logs relating to the provided folder are included."
```

Comment:
```suggestion
  description = "(Optional) Whether or not to include child projects in the sink export. If true, logs associated with child projects are also exported; otherwise only logs relating to the provided folder or organization are included."
```